src/CVSelect/ContentViewList.js

```diff
@@ -39,7 +39,19 @@ export default function ContentViewList() {
     window.location.assign(`/content-view/${id}`);
   };
 
-  const handleOnSearchChange = () => {};
+  const handleOnSearchChange = (val) => {
```

Comment:
I can get you more info on this, but this should be "debounced". Right now on each keypress you run the search. That is a lot of extra rendering. Debouncing allows you to set a time delay. It waits until there is X amount of time after the last key press, and then will execute. This reduces unnecessary renders.